src/main.rs

```diff
@@ -219,10 +219,10 @@ mod tests {
         false
     }
 
-    #[test]
     #[allow(clippy::unit_cmp)]
+    #[test]
     fn test_main() {
-        const FIRECRACKER_INIT_TIMEOUT_MILLIS: u64 = 100;
+        const FIRECRACKER_INIT_TIMEOUT_MILLIS: u64 = 150;
```

Comment:
I'm OK with the extra 50 ms but I never saw this causing a failure

Reply:
It's only failing with Rust 1.35 when running under kcov. When running `cargo test` it is not failing.